### DISCUSSION TITLE
我第一次打开alist报错，我的理解是tcp5244端口被占用，后来百度，添加了防火墙出站规则，重启后成功了，但是第二天有事同样的报错，可是防火墙规则还在，这怎么解决？

### DISCUSSION THREAD

**Discussion by tgfydzs** [AlistGo/alist#1616] [Q&A]
我第一次打开alist报错，我的理解是tcp5244端口被占用，后来百度，添加了防火墙出站规则，重启后成功了，但是第二天有事同样的报错，可是防火墙规则还在，这怎么解决？

**Comment by tgfydzs** [comment]
具体报错码： failed to start: listen tcp 0.0.0.0:5244: bind: An attempt was made to access a socket in a way forbidden by its access permissions.

**Reply by IMXeno123** [reply]
这个错误似乎与端口占用无关

**Reply by xinxinenjoy** [reply]
防火墙规则只能阻挡局域网访问它的数据，报错应该是别的原因

**Reply by tbsmqx** [reply]
是不是百度开着,他会占用5244端口

**Comment by silent-wind1** [comment]
访问出现报错：System error: SyntaxError: Invalid regular expression: /?/: Nothing to repeat是什么问题

**Comment by miao5748** [comment]
netsh winsock reset

**Comment by xinxinenjoy** [comment]
端口占用应该就是字面意思，这个端口已经有别的服务在运行了
出站规则只能阻止你局域网其他设备来访问这个服务，并不会阻止本机访问
